### DISCUSSION TITLE
Can the online editor be used instead or Sublime?

### DISCUSSION THREAD

**Discussion by mikelgiles** [elm-time/elm-time#22] [Q&A]
I am new to elm but really like it a lot and a big part of that is due to the online editor. I was very dissapointed in the offline methods I was pointed too.  Just wondering if it is possible continue with the online editor? I am very old and retired but have a financial app that was written in VB6 and needs out of it because maintaining VB6 is so hard. I want to move it to the internet.

**Comment by Viir** [comment]
Hey Mikel, welcome to the Elm community!

You certainly can continue with the online editor unless you need functionality not yet available there. It depends a bit on the specifics of your project.
There are some Elm-related tools out there that are not yet integrated with Elm Editor. For example, the support for automated tests using an entry point of type `Test.Test` is still a work in progress and unavailable today. Another example: If you have more than twenty Elm modules in your project, a local development setup might provide much faster response times than the current version of the online editor (for example, for features like completion suggestions or hover tips). The work to improve runtime efficiency and response times is ongoing.
As far as I understand your project, there is no need to invest in maintaining a local development environment.

If you hit any bumps or have questions, don't hesitate to share. We're here to help and grow together. Happy coding! 🚀

**Reply by mikelgiles** [reply]
What do you know about the online editor? I never figured out how to use it other than access thru the official tutorial.  I found several on Github but didn't research any of them. I downloaded sublime and looked at it and all of the simple of ELM just went away. Do you know how to use the online editor separated from the tutorial?

**Reply by Viir** [reply]
You can use the online editor by navigating to https://elm-editor.com/ in your web browser. That site is a public instance that is free to use for everyone. You can import your project/workspace and work there and export the workspace to make backups.

Also, the `Load Project from Git Repository` menu entry lets you import a workspace directly from a public repository on GitHub.
